src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -642,61 +644,32 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
   const renderShowScheduleToggle = () => (
```

Comment:
```suggestion
  // @debt pass the header into Toggler's 'label' prop instead of being external like this
  const renderShowScheduleToggle = () => (
```